cclib/parser/nwchemparser.py

```diff
@@ -359,11 +370,15 @@ def extract(self, inputfile, line):
                 for attr in scftargetattrs:
                     delattr(self, attr)
 
-        #DFT functional information
+        # DFT functional and basis set information
         if "XC Information" in line:
             line = next(inputfile)
             line = next(inputfile)
             self.metadata["functional"] = line.split()[0]
+        
+        if "Grid Information" in line:
+            line = next(inputfile)

```

Comment:
This isn't right.  The section looks like
```
             Grid Information
             ----------------
          Grid used for XC integration:  xfine
          Radial quadrature: Mura-Knowles
          Angular quadrature: Lebedev.
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          C                   0.70      100          13.0      1454
          H                   0.35      100          13.0      1202
          Grid pruning is: on
          Number of quadrature shells:  1000
          Spatial weights used:  Erf1
```
so this just takes the dashes.  We need a `skip_line` on dashes.